cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4157,4 +4157,11 @@
       Internal directory that maintains the currently disabled and enabled configurations
     </description>
   </property>
+  <property>
+    <name>capability.autoinstal.threads</name>
```

Comment:
s/autoinstal/autoinstall here and below.

Reply:
thanks, fixed.